src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -9,6 +9,7 @@ <h4>{{ data.name }} </h4>
                         {{ messagesKey['contenttypes.content.variable'] }}: {{ data.variable }}
                         <span class="content-type__edit-dot-separator">•</span> {{ messagesKey['contenttypes.form.identifier'] }}: {{ data.id }}
                     </span>
+                    <dot-api-link href="/api/v1/contenttype/id/{{data.id}}"></dot-api-link>
```

Comment:
maybe we should have something like: dot-api-content-type-link, it could be a wrapper of dot-api-link, and just receive the content type or content type's id, in this way we encapsulate this url: /api/v1/contenttype/id/{{data.id}}

Reply:
Why? so far this is the only place in content type where we going to use this, don't see the need at this point.